ui/Screen/Org/Projects.svelte

```diff
@@ -41,20 +57,65 @@
     max-width: var(--content-max-width);
     min-width: var(--content-min-width);
   }
+
+  .pending {
+    margin-bottom: 2rem;
+  }
+
+  .header {
+    display: flex;
+    padding: 0.5rem 3rem 0.5rem;
+    width: 100%;
+  }
 </style>
 
 <div class="container">
-  {#if anchoredProjects.length !== 0 || unresolvedAnchors.length !== 0}
-    <ProjectList
-      projects={anchoredProjects}
-      userUrn={session.identity.urn}
-      on:select={select} />
-
-    <UnresolvedAnchorList anchors={unresolvedAnchors} />
+  {#if pendingResolved.length !== 0 || pendingUnresolved.length !== 0}
+    <div class="pending">
+      <div class="header">
+        <p class="typo-text-bold">Pending</p>
+        <p style="margin-left: .5rem; color: var(--color-foreground-level-6);">
+          Not enough of the members have signed this anchor.
+        </p>
+      </div>
+      {#if pendingResolved.length !== 0}
+        <ProjectList
+          projects={pendingResolved}
+          userUrn={session.identity.urn}
+          on:select={select} />
+      {:else if pendingUnresolved.length !== 0}
+        <UnresolvedAnchorList anchors={pendingUnresolved} />
+      {/if}
+    </div>
+  {/if}
+  {#if confirmedResolved.length !== 0 || confirmedUnresolved.length !== 0}
+    {#if pendingResolved.length !== 0 || pendingUnresolved.length !== 0}
```

Comment:
There's a logic bug here. If there are no confirmed anchors but there are pending ones, then the pending ones would not be shown. Fixed in 5518ed210d262ff6090d31f4369114c76e471e3a.

Reply:
Hmm, I was trying to show the header only when there are pending anchors. So that when there are no pending, there is also no title.